sofascoreAcademy/sofascoreAcademy/LoginViewController.swift

```diff
@@ -0,0 +1,222 @@
+        titleLabel.backgroundColor = .clear
+        titleLabel.textColor = .primaryBlack
+        titleLabel.textAlignment = .center
+        titleLabel.font = .headlineBold32
+        titleLabel.numberOfLines = 2
+        
+        textLabel.text = "Hundreds of stats. Infinite love of the game."
+        textLabel.backgroundColor = .clear
+        textLabel.textColor = .primaryBlack
+        textLabel.textAlignment = .center
+        textLabel.font = .regular14
+      
+        usernameTextField.placeholder = "Username"
+        usernameTextField.backgroundColor = .clear
+        usernameTextField.borderStyle = .roundedRect
+        usernameTextField.font = .regular14
+        usernameTextField.textColor = .secondaryGray
+        usernameTextField.autocapitalizationType = .none
+                
+        passwordTextField.placeholder = "Password"
+        passwordTextField.backgroundColor = .clear
+        passwordTextField.borderStyle = .roundedRect
+        passwordTextField.isSecureTextEntry = true
+        passwordTextField.font = .regular14
+        passwordTextField.textColor = .secondaryGray
+        passwordTextField.autocapitalizationType = .none
+        
+        loginButton.setTitle("Login", for: .normal)
+        loginButton.backgroundColor = .headerBackground
+        loginButton.layer.cornerRadius = 10
+        loginButton.addTarget(self, action: #selector(loginButtonTapped), for: .touchUpInside)
+                
+        stackView.axis = .vertical
+        stackView.backgroundColor = .clear
+        stackView.distribution = .fillEqually
+        stackView.alignment = .fill
+        stackView.spacing = 0
+        
+        errorLabel.textColor = .red
+        errorLabel.backgroundColor = .clear
+        errorLabel.font = .regular14
+        errorLabel.textAlignment = .center
+        errorLabel.text = ""
+        errorLabel.numberOfLines = 2
+    }
+    
+    private func updateLayoutForOrientation() {
+        if UIDevice.current.orientation.isLandscape {
+            stackView.axis = .horizontal
+            textContainerView.snp.remakeConstraints {
+                $0.width.equalTo(view.safeAreaLayoutGuide).multipliedBy(0.5)
+                $0.leading.trailing.equalTo(view).inset(20)
+            }
+            videoContainerView.snp.remakeConstraints {
+                $0.width.equalTo(view.safeAreaLayoutGuide).multipliedBy(0.5)
+                $0.top.bottom.equalTo(view).inset(16)
+            }
+        } else {
+            stackView.axis = .vertical
+            textContainerView.snp.remakeConstraints {
+                $0.height.equalTo(view.safeAreaLayoutGuide).multipliedBy(0.5)
+                $0.leading.trailing.equalTo(view).inset(20)
+            }
+            videoContainerView.snp.remakeConstraints {
+                $0.height.equalTo(view.safeAreaLayoutGuide).multipliedBy(0.5)
+                $0.leading.trailing.equalTo(view).inset(16)
+            }
+        }
```

Comment:
Lijepo što brineš za ovo, ali možeš zanemariti horizontalni landscape ako ti stvara probleme, ili disableati u Xcodeu settingsima... u svakom slučaju dobro ga za test koristiti jer onda vidiš jel dobro UI postavljen.